docs/location-strategies.adoc

```diff
@@ -179,13 +179,17 @@ public class TestClassOrPageObjectOrPageFragment {
   private List<MyPageFragmentImpl> myListOfPageFragments;
 
   @FindBy(id = "select")
-  private Select select;
+  private GrapheneSelect select;
```

Comment:
If we advertise this as a solution maybe it's worth moving it to some other module, instead of bundling it in `-test.jar`?

Reply:
+1